modules/_node-scripts/bin.js

```diff
@@ -32,7 +32,7 @@ const COMMANDS = {
 	'format': {
 		description:
 			'formats source files or optionally only checks with "--check" flag.',
-		parameters: '[--check]',
+		parameters: '[--all, --check]',
```

Comment:
This should be something like `[--all] [--check]` if both are optional and both can be given at the same time.

`[{--all|--check}]` if they are optional but exclusive, etc

Based on the informal unix/DOS command line syntax documentation